### PR TITLE
Pull for Refactoring

### PR DESCRIPTION
**Pull request by spina-a-d**
Sorry its slightly late, I had assumed from the chat other members of the team were handling it and didn't realize they hadn't.  Hope 8 minutes isn't too late.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,18 @@ Team Wander’s aim is to create a cognitive testing game which will use existin
...
 
 ## App
-The Android application files are located in /Wander. The application was developed and can be built using Android Studio.
+The Android application files are located in 2017-Cognitive-Sensors/Wander. The application was developed and can be built using Android Studio.  In order to do this one simply clones the repository to their computer.  Then, checkout the branch that work will be done in.  Open Android Studio (make sure its fully updated or there might be build errors) and navigate to the File - > Open... -> and navigate to the Wander folder (which should have an Android Studio icon indicating that it is openable as a project) and open it.
+
+If you've never used Android Studio before there are a few things to note.  Access files from the project on the far left set of tabs under "Project" and then all the code is available, and the Java code specifically under the package that is NOT labelled (androidTest) or (test).  In order to test code you should configure a virtual phone and run it on there.  You can also put your personal android phone in developer mode and test code on there which is a lot quicker to setup.
```
Comment (PJUllrich) [inline README.md:9]: Add links here to websites that explain how to 'configure a virtual phone' and how to 'put your personal android phone in developer mode'

File: README.md [hunk 2]
```diff
@@ -4,9 +4,18 @@ Team Wander’s aim is to create a cognitive testing game which will use existin
...
+
+Something to note: If you're not completely up to date with your Android Studio, particularly Gradle, your builds might encounter issues.  To avoid this please just update everything before you even clone the repository (if you've just downloaded the software suite you're fine).  Also, you must wait for the Gradle build to finish before you can run the app, this can take a few minutes the first time you open it.
```
Comment (PJUllrich) [inline README.md:11]: Explain how to update the Android Studio (assume the person cloning this repository is a monkey)

File: README.md [hunk 3]
```diff
@@ -4,9 +4,18 @@ Team Wander’s aim is to create a cognitive testing game which will use existin
...
 
-![JSON structure accepted by web app. This is also available in the architecture document.](https://raw.githubusercontent.com/RUGSoftEng/2017-Cognitive-Sensors/feature-readme/Server/JSON_structure.png)
+![JSON structure accepted by web app. This is also available in the architecture document.](Server/JSON_structure.png)
+
+## Setup
+
+### Server
+To set up the server, first create a new Google Spreadsheet. Add four sheets to the spreadsheet, called *GameSessions*, *NumberGuesses*, *QuestionAnswers* and *Questions*. The data will be stored in these sheets. Now open the scripteditor (*Extra*, *Scripteditor…*). Add the scripts in the folder [Server](Server) to the project. To deploy the app, click *Publish* and then *Implement as web app…*. This will give you a URL you can use for your application.
```
Comment (PJUllrich) [inline README.md:21]: Where do you have to put that URL in the code?

**Review by PJUllrich — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Lolmewn) [reply README.md:11]: I am a monkey and I want to clone this repository.

Comment (PJUllrich) [reply README.md:11]: Haha! Sybren, why are you in this repository?! :D

Comment (spina-a-d) [reply README.md:11]: @PJUllrich @Lolmewn  Ok thanks for the feedback, I'll try and make it even clearer.

**Pull request merged**